nengo_viz/templates/page.html

```diff
@@ -11,15 +11,15 @@
         <link href="static/viz_tooltips.css" rel="stylesheet" type="text/css" />
         <link href="static/viz_sim_control.css" rel="stylesheet" type="text/css" />
         <link href="static/viz_netgraph.css" rel="stylesheet" type="text/css" />
+        <link href="static/viz_ace.css" rel="stylesheet" type="text/css" />
         <link href="static//lib/css/bootstrap.min.css" rel="stylesheet" type="text/css" />
         <link href="static/viz_top_toolbar.css" rel="stylesheet" type="text/css" />
         <link href="static/lib/css/bootstrap.min.css" rel="stylesheet" type="text/css" />
         <link href="static/lib/js/jqueryFileTree/jqueryFileTree.css" rel="stylesheet" type="text/css">
     </head>
-
     <body onload="onload();">
-        <div id="top_toolbar">
-            <ul class="nav nav-pills">
+        <div id="top_toolbar_div">
```

Comment:
> Why make this change? Seems overly verbose...

I believe that this was a result of some debugging/testing where there was also a top_toolbar id in existence.